### ISSUE TITLE
Изменить размер основного контейнера для мобильной версии

### ISSUE THREAD

**Issue by ehassed3** [VladimirBulanov/VladimirBulanov#4]
Для десктопа и планшета текст читаем, но для мобильной версии приходится использовать приближение.

**Comment by BekoBou** [comment]
Контейнер имеет ширину 100% — это не поможет. Здесь другая проблема.

**Comment by ehassed3** [comment]
Да, сформулировал неправильно — имел ввиду внутреннее содержимое. Как вариант решения можно увеличить размер шрифта.

**Comment by BekoBou** [comment]
Это не решение проблемы. Задача будет решена, но это будет очень плохое решение, которое потом ещё и сопровождать неудобно.
